client/src/components/modals_and_popovers/FixedPopover.tsx

```diff
@@ -43,22 +67,27 @@ export class FixedPopover extends React.Component {
     document.body.classList.remove("modal-open--allow-scroll");
     this.setState({ timeout_stopped: false });
 
-    this.props.on_close_callback();
+    const { on_close_callback = _.noop } = this.props;
+
+    on_close_callback();
   };
+
   render() {
     const {
+      // if the popup gets too tall, it will be cut-off (and possibly non-interactable for it) on mobile
+      // 40vh is a bit arbitrary as a default, but leaves room for long header/footer content
+      max_body_height = "40vh",
       show,
       title,
       subtitle,
       header,
       body,
-      max_body_height,
       footer,
-      dialog_position,
+      dialog_position = "left",
       additional_dialog_class,
-      auto_close_time,
-      close_text,
-      close_button_in_header,
+      auto_close_time = false,
+      close_text = _.upperFirst(trivial_text_maker("close")),
+      close_button_in_header = false,
```

Comment:
Shouldn't `on_close_callback = _.noop` (or equivalent) go here? Why is it up on line 70?

Did TS complain about the type of `_.noop` or something?